### PR TITLE
[Task-45]-Resize button and alignment in all the components

### PR DESCRIPTION
**Pull request by rajiravenantech**
Resize button and alignment in all the components

### CODE REVIEW

File: src/app/components/BasicUIElements/ButtonComponent.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  name: string;
+  disabled?: boolean;
+  type?: "button" | "submit" | "reset" | undefined;
+  style?: React.CSSProperties;
+  startIcon?: React.ReactNode;
+  size?: "small" | "medium" | "large" | undefined;
+  color?: "secondary" | "info" | "success" | "warning" | "primary" | undefined;
+}
+
+const ButtonComponent: React.FC<ButtonComponentProps> = ({
```
Comment (esakkirajaravenantech) [inline src/app/components/BasicUIElements/ButtonComponent.tsx:15]: Component name need to change as `Button` and startIcon should be prefix and add suffix also.

File: src/app/components/ButtonElement/OutlineButtonElement.tsx [hunk 2]
```diff
@@ -9,24 +10,43 @@ const OutlineButtonElement = () => {
...
-          Info
-        </Button>
-        <Button variant="outlined" color="primary" disabled>
-          Disabled
-        </Button>
+        <ButtonComponent
+          variant="outlined"
+          color="primary"
+          name="Primary"
+          style={{ borderRadius: "10px" }}
```
Comment (esakkirajaravenantech) [inline src/app/components/ButtonElement/OutlineButtonElement.tsx:17]: No need to pass `style={{ borderRadius: "10px" }}` in every use,  make 10px is default value incase not passing props in button component.

File: src/app/components/BasicUIElements/ButtonComponent.tsx (hunk 1 above)
Comment (rajiravenantech) [reply src/app/components/BasicUIElements/ButtonComponent.tsx]: Changed the button Component name.  Added prefix and suffix props in button component.

File: src/app/components/ButtonElement/OutlineButtonElement.tsx (hunk 2 above)
Comment (rajiravenantech) [reply src/app/components/ButtonElement/OutlineButtonElement.tsx]: Modified the border radius as default style

**Review by esakkirajaravenantech — APPROVED** [review]

**Pull request merged**